### PR TITLE
Content and accessibility Sweep - RIP-235 

### PR DESCRIPTION
**Pull request by autotelik**
https://eaflood.atlassian.net/browse/RIP-235

### CODE REVIEW

**Comment by autotelik** [comment]
:eyes:

File: app/assets/stylesheets/partials/_footer.scss [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
-    text-transform: none;
-  }
-
-  .inline {
-    display: inline;
+}
+@media (min-width: 641px){
+  .ea-footer {
+    padding-right: 30px;
+    font-size: 16px; // TODO use preset
```
Comment (woodpigeon) [inline app/assets/stylesheets/partials/_footer.scss]: What does `// TODO use preset` mean?

Comment (autotelik) [inline app/assets/stylesheets/partials/_footer.scss]: not sure .. Martin supplied this .. I'll check

Comment (autotelik) [inline app/assets/stylesheets/partials/_footer.scss]: "It means the SASS should use a variable, but I did not have time to sort it...
  SASS removes comments doesn’t it?"

Comment (woodpigeon) [inline app/assets/stylesheets/partials/_footer.scss]: There are scss variables in e.g govuk_frontend_toolkit that you could use but I had a look and its not very obvious which ones - so I would just delete the comments

Comment (autotelik) [inline app/assets/stylesheets/partials/_footer.scss]: how about i change it to read :

 // TODO - can we use the preset variables in place of these hard coded pixels ?

else Martin happy if we just delete it

File: app/assets/stylesheets/partials/_forms.scss [hunk 2]
```diff
@@ -19,24 +19,25 @@ span.postcode {
...
 
-span.exmpt {
-  float: right;
-  font-weight: bold;
-  margin-right: -3em;
+
+span.exmpt
+{
+  font-weight:bold; float:right; margin-right:-3em;
 }
```
Comment (woodpigeon) [inline app/assets/stylesheets/partials/_forms.scss]: Rules should be on separate lines

File: app/assets/stylesheets/partials/_forms.scss [hunk 3]
```diff
@@ -19,24 +19,25 @@ span.postcode {
...
 
-label.exemption {
-  background-color: #ededed;
-  border-color: #ededed;
-  padding-right: 4em;
+label.exemption
+{
+  background-color:#ededed; border-color:#ededed; padding-right: 4em;
```
Comment (woodpigeon) [inline app/assets/stylesheets/partials/_forms.scss]: Rules should be on separate lines

File: app/assets/stylesheets/partials/_forms.scss [hunk 4]
```diff
@@ -19,24 +19,25 @@ span.postcode {
...
 }
 
-@media (min-width: 641px) {
-  label.exemption {
-    width: 99%;
-  }
+@media screen and (min-width: 320px)
+{
+  label.exemption
+  { width: 66.6666666667%; }
```
Comment (woodpigeon) [inline app/assets/stylesheets/partials/_forms.scss]: formatting out a bit ```abel.exemption
-  }
+  { width: 66.6666666667%; }

File: app/assets/stylesheets/partials/_footer.scss (hunk 1 above)
Comment (woodpigeon) [inline app/assets/stylesheets/partials/_footer.scss]: Your call I don't mind - if comments removed then all good.

**Comment by woodpigeon** [comment]
Couple of css layout comments, otherwise :ship: it

**Pull request merged**